src/components/BasePaymentState.vue

```diff
@@ -7,6 +7,26 @@
         <div class="text-center" v-if="payment.state === PaymentState.Completed">
             <i class="fas fa-thumbs-up text-success mb-3" style="font-size: 3rem;"></i>
             <p class="text-gray"><strong>THX!</strong> This payment has been completed.</p>
+            <div class="d-flex justify-content-between" v-if="payment.metadata">
+                <div>
+                    <p class="text-gray">The <strong>NFT</strong> has been minted</p>
+                </div>
+                <div class="text-gray">
+                    {{ payment.metadata.title || 'No Title' }}
```

Comment:
@valeriagrazzini Recently learned that in this case it's more semantically correct to use `??` instead of `||`. 

> The OR operator (double pipe) || uses the right value if the left is falsy, while the nullish coalescing operator ?? (double question mark) uses the right value if the left is null or undefined .

Reply:
If I use 
{{ payment.metadata.title ?? 'No Title' }}
i get a compilation error:
Syntax Error: SyntaxError: Unexpected token (1:650)

Reply:
Ah then the compiler probably is not configured for ES2020 I guess, no worries